hack/terraform/azure/main.tf

```diff
@@ -28,7 +28,7 @@ resource "azurerm_resource_group" "vm" {
 
 locals {
   resource_group_name = "nephe-vnet-${var.owner}-${random_string.suffix.result}"
-  azure_vm_os_types   = var.with_agent ? var.azure_vm_os_types_agented : var.azure_vm_os_types
+  azure_vm_os_types   = var.with_agent ? (var.with_windows ? var.azure_win_vm_os_types_agented : var.azure_vm_os_types_agented) : var.azure_vm_os_types
```

Comment:
please change the constant to azure_vm_os_types_agented_windows

Reply:
Done